packages/protoplugin-example/src/protoc-gen-twirp-es.ts

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env npx tsx
+
+import { createEcmaScriptPlugin } from "@bufbuild/protoplugin";
+import { generateTs } from "./typescript.js";
+import { version } from "../package.json";
+import { runNodeJs } from "@bufbuild/protoplugin";
+
+export const protocGenTwirpEs = createEcmaScriptPlugin({
```

Comment:
Importing this file runs the plugin, so we should not export this anymore. 

```suggestion 
const protocGenTwirpEs = createEcmaScriptPlugin({
```